src/main/java/com/manywho/services/dummy/dummy/Account.java

```diff
@@ -0,0 +1,43 @@
+package com.manywho.services.dummy.dummy;
```

Comment:
This class would be better organized in other domain directory like `com.manywho.services.dummy.account`